tests-invoke

```diff
@@ -49,7 +49,8 @@ def main():
 
         if pull:
             if pull["head"]["sha"] != opts.revision:
-                return f"Newer revision {pull['head']['sha']} available on GitHub for this pull request"
+                return (f"Newer revision {pull['head']['sha']} available on GitHub for this pull request;"
+                        f"node_id {pull['node_id']}, updated_at {pull['updated_at']}")
```

Comment:
This now returns a tuple which is fed into .format() which only has a {0}, so I think this outputs a tuple or is that intended?